draft-ietf-scitt-architecture.md

```diff
@@ -373,280 +369,155 @@ Most of the details of the Receipt's contents are specified in the COSE Signed M
+During initialization of a Transparency Service, the first Signed Statements registered will be for a trust anchor that is not validated by any Registration Policy.
 
-There is no fork in the Append-only Log.
-Everyone with access to its contents sees the same sequence of entries, and can check its consistency with any Receipts they have collected.
-Transparency Service implementations MAY provide a mechanism to verify that the state of the Append-only Log, encoded in an old Receipt, is consistent with the current Append-only Log state.
-
-#### Replayability and Auditing
-
-Everyone with access to the Transparency Service can check the correctness of its contents.
-In particular:
+Transparency Services MUST specify their supported signature algorithms in their Registration Policies.
 
-- the Transparency Service defines and enforces deterministic Registration Policies that can be re-evaluated based solely on the contents of the Append-only Log at the time of Registration, and must then yield the same result
-- the ordering of entries, their cryptographic contents, and the Transparency Services' governance may be non-deterministic, but they must be verifiable
-- a Transparency Service MAY store evidence about the resolution of identifiers, identity documents, and key material.
-- a Transparency Service MAY additionally support verifiability of client authentication and access control
+This specification leaves implementation and encoding of Registration Policy to the operator of the Transparency Service.
 
-#### Governance and Bootstrapping
+### Append-only Log
 
-Transparency Services MAY document their governance rules and procedures for operating the Transparency Service and updating its code.<br>
-Example: relying on Transparent Statements about code updates, secured on its own Append-only Log, or on some auxiliary Transparency Service.<br>
+The security properties of the Append-only Log are determined by the choice of the verifiable data structure used to produce Receipts.
 
-Governance procedures, their auditing, and their transparency are implementation specific.
+In addition to Receipts, some verifiable data structures might support additional proof types, such as proofs of consistency, or proofs of non inclusion.
 
-- Governance may be based on a consortium of members that are jointly responsible for the Transparency Services, or automated based on the contents of an auxiliary governance Transparency Service.
-- Governance typically involves additional records in the Append-only Log to enable its auditing.
-The Transparency Service may contain both Transparent Statements and governance entries.
-- Issuers, Verifiers, and third-party Auditors may review the Transparency Service governance before trusting the service, or on a regular basis.
+Specific verifiable data structures, such those describes in {{-CT}} and {{-COMETRE}} are out of scope for this document.
```

Comment:
But at least their intended security properties are? If this is out of scope of SCITT then at least the COMETRE document must be much more specific about the minimum security properties of verifiable data structures.

Reply:
Yes. That is a COMETRE task (or other Append-only Log specification text).